modules/apps/asset/asset-categories-selector-web/src/main/resources/META-INF/resources/js/SelectCategory.es.js

```diff
@@ -39,21 +39,23 @@ function SelectCategory({
 	nodes,
 }) {
 	const flattenedNodes = useMemo(() => {
-		if (nodes.length === 1 && nodes[0].vocabulary) {
+		if (nodes.length === 1 && nodes[0].vocabulary && nodes[0].id !== '0') {
```

Comment:
I don't really get this change, what's the reason for it?

Reply:
The view look like this:

<img width="1448" alt="Screenshot 2020-11-04 at 14 45 56" src="https://user-images.githubusercontent.com/11654230/98121132-2db6d900-1eaf-11eb-9692-e6301e7d263a.png">

I've changed the model to add the property vocabulary: true to the root node, but doing so this code is removing that root, so I've just had to add this check here to preserve the root after setting the property

Another option would be to check for id !== '0' in the filter function, but I think it isn't much of an improvement

Also I am not sure where this code is used, I've seen that this code can be improved a lot, but I've wanted to make minimal changes here in order to fix the bug, and revisit this later, (when removing aui stuff for example)